vagrant-pxe-harvester/Vagrantfile

```diff
@@ -81,25 +81,27 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   # Rancher
-  config.vm.define :rancher do |rancher|
-    rancher.vm.box = 'generic/debian10'
-    rancher.vm.hostname = 'rancher'
-    rancher.vm.network 'private_network',
-      ip: @settings['rancher_config']['ip'],
-      libvirt__network_name: 'harvester',
-      libvirt__dhcp_enabled: false
+  if @settings['rancher_config']['enabled'] == true
```

Comment:
I'm not familiar with ruby, just wondering if we could simplify with:
```
if @settings['rancher_config']['enabled']
```

Reply:
This will work, too.